packages/contentful--create-contentful-app/src/app-actions/javascript/build-actions.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs');
+const { readdirSync, readFileSync, writeFileSync, mkdirSync } = fs;
+
+const buildActions = () => {
+  const actionsFiles = readdirSync('actions', {
+    encoding: 'utf-8',
+  });
+
+  const actions = actionsFiles.map((fileName) => {
+    return readFileSync(`actions/${fileName}`, {
+      encoding: 'utf-8',
+    }).toString();
+  });
+
+  mkdirSync('build/actions', { recursive: true });
+
+  actionsFiles.map((fileName, index) => {
+    return writeFileSync(`build/actions/${fileName}`, actions[index]);
```

Comment:
you can merge these two `map`s into one and replace them with `forEach`. There's no need in map.

Reply:
Ah I was doing this async originally, and I think that's why the maps are there. I'll change that :)